src/editorIdentity.ts

```diff
@@ -13,11 +13,15 @@ export class EditorIdentity {
   /**
    * For use in tests
    */
-  public static createRandomEditorIdentity(): EditorIdentity {
+  public static createRandomEditorIdentity(
```

Comment:
Why not just call `new EditorIdentity` in the test yourself? Seems weird to call `createRandom` and then override the randomness

Reply:
the constructor for `EditorIdentity` is marked as `private`. Should i made it `public` or create a new method for it?

Reply:
Just make it public; you're exposing essentially the same API but less clearly by adding a parameter to the create random function.

Reply:
Sure, i also removed the `createRandomEditorIdentity` method.